src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -30,6 +30,6 @@ const GroupDiv = styled.div`
  */
 export const ButtonGroup = ({
   children,
-}: ButtonGroupProps): React.ReactNode => {
+}: ButtonGroupProps): JSX.Element => {
```

Comment:
Need to update docstring here, but I'm wondering what's the difference here.

Also might want to update line 9 as well, but I think children should be a list of the type `Button`, correct me if I'm wrong.

Reply:
Typescript has some specific guidelines as to what children can be typecasted to. A list of `Button` components falls under the type `React.ReactNode`. Using `children` instead of a different variable name makes it easier when using the `ButtonGroup`.  

If we were to make the type more specific we would have to use a different variable name. It would like the following:

```
<ButtonGroup buttons={[<Button></Button>, <Button></Button>]}/>
```

This is what it would look like using children
```
<ButtonGroup>
    <Button></Button>
    <Button></Button>
</ButtonGroup>